src/main/java/com/nguyenquyhy/discordbridge/utils/ColorUtil.java

```diff
@@ -65,4 +68,11 @@ public static TextColor getColor(Color color) {
         }
         return result;
     }
+    public static String removeColor(String text){
```

Comment:
As per Issue #50 , Messages that included Minecraft color codes such as &4 and &3, were getting sent to discord. Its now checking every line sent to discord and replacing any of the Minecraft color codes with "", effectively removing them.